docs/spacing.md

```diff
@@ -0,0 +1,93 @@
+# Spacing
+
+Careful spacing is key for the user to understand connections and hierarchies between elements, and for an interface to feel pleasant.
+
+## Scale
+
+Primer follows a base-8 scale, as it allows for significant flexibility when defining a variety of spacing values within an interface.
+
+`[list of spacing utilities]`
+
+For very small components where 8 pixel spacing is too large, you can use 4 pixels. 
+
+## Responsive breakpoints
+
+You can define different spacing values for different breakpoints, so the interface can better adapt to the available space.
+
+`[Example: component with larger margin around it in large screen, vs smaller margin in small screen]`
+
+In code, margins can be defined per breakpoint, where different breakpoints have a different margin applied.
+
+`[Example: mb-sm-2 mb-md-3 mb-lg-4]`
+
+## Applying spacing
+
+The general rule when using Primer is to apply spacing in increments of 8 pixels, as provided by the spacing utilities. 
+
+In some cases, for very small components and in denser areas of the UI, you can use 4 pixels.
+
+`[Example: text and icon button]`
+
+Primer components have inner spacing built-in, so in most cases you’ll only need to consider vertical spacing between components.
+
+Do/Don’t: Using custom values?
+
+## Relationships
+
+Elements that are close together are perceived as being related or part of the same group, while elements that are far apart from each other feel unrelated.
+
+`[Examples: cancel, save buttons; repo top nav buttons]`
+
+Alignment
+- Do / Don’t
```

Comment:
How do we handle inset alignment across components? For example, does the header of the table align with the row content? Or are the edges of the bounding box that align?